internal/provider/data_source_kubernetes_clusters.go

```diff
@@ -0,0 +1,286 @@
+package provider
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha1"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"wiz.io/hashicorp/terraform-provider-wiz/internal"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/client"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/utils"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/vendor"
+)
+
+// ReadKubernetesClusters struct
+type ReadKubernetesClusters struct {
+	KubernetesClusters vendor.KubernetesClusters `json:"kubernetesClusters"`
```

Comment:
Refactor the type from KubernetesClusters to KubernetesClusterConnection.

```suggestion
	KubernetesClusters vendor.KubernetesClusterConnection `json:"kubernetesClusters"`
```